TappNetwork/home-visiting-app-web
# Ios build

# Details

This PR encompasses changes for the ios build as well as fixes for UI bugs that were noticed when playing with the phone build in the simulator

# Steps to Test
1. Build ios app with 'ionic cap run'
2. Make sure Main header does not overlap any content
3. Make sure title sizes are properly responsive and never are cut off or truncated strangely
4. Test slide out navbar for functionality and no weird display glitches, especially with corners being rounded on mobile
5. The cards on the dashboard should now render much better on mobile without excessive padding
6. Page headers should no longer have white backgrounds but should instead match the rest of the site

# Tasks Linked 
[2m9dq2a](https://app.clickup.com/t/2m9dq2a)

# Screenshots
<img width="386" alt="Screen Shot 2022-09-26 at 2 53 48 PM" src="https://user-images.githubusercontent.com/29612767/192378640-aa31d89a-12e9-452d-86f9-8183dbe49570.png">
<img width="393" alt="Screen Shot 2022-09-26 at 2 54 46 PM" src="https://user-images.githubusercontent.com/29612767/192378654-638fd3ae-d342-4b02-b9cf-fc9deea14f0b.png">
<img width="390" alt="Screen Shot 2022-09-26 at 2 53 58 PM" src="https://user-images.githubusercontent.com/29612767/192378660-099be5af-1ac3-44c9-9880-246f538207c1.png">
<img width="391" alt="Screen Shot 2022-09-26 at 2 54 05 PM" src="https://user-images.githubusercontent.com/29612767/192378667-2f78df73-e452-4607-8ea0-f2218e1e8566.png">
<img width="394" alt="Screen Shot 2022-09-26 at 2 54 27 PM" src="https://user-images.githubusercontent.com/29612767/192378670-2250c6e6-b062-4f6f-a6c4-910e945f9fd3.png">
